src/main/java/carRacing/Car.java

```diff
@@ -0,0 +1,22 @@
+package carRacing;
+
+public class Car {
+
+	private static final int MINIMUM_TO_MOVE = 4;
+
+	private int position;
+
+	public void moveCar(int randomNum) {
+		if (isCarFoward(randomNum)) {
+			position++;
+		}
+	}
+
+	public boolean isCarFoward(int num) {
+		return num >= MINIMUM_TO_MOVE;
+	}
```

Comment:
```suggestion
	private boolean isCarForward(int num) {
		return num >= MINIMUM_TO_MOVE;
	}
```
moveCar에서만 사용하고 있으니 private 접근제한자로 변경하는 게 좋겠네요